packages/styles/scss/components/number-input/_number-input.scss

```diff
@@ -436,17 +436,43 @@
   }
 
   // Readonly
+  // placeholder should be moved to tokens
+  $icon-readonly: $icon-disabled;
```

Comment:
@carbon-design-system/design Should this be a separate token, or just use `$icon-disabled`?

Reply:
@tay1orjones removed $icon-disabled and raised a question #12444